docs/content/users/usage/commands.md

```diff
@@ -337,12 +337,12 @@ ddev debug dockercheck
 
 ### `debug download-images`
 
-Download all images required by DDEV.
+Download the basic docker images required by DDEV. This can be useful on a new machine to prevent `ddev start` or other commands having to download the various images.
```

Comment:
You understand completely. `ddev debug` is just a place to dump commands that people shouldn't normally need. This one is really useful and will help the Gitpod setup, because it currently requires a project to do its work, and that's a waste. And yes, it may find a way to become part of the install. It would be nice if the image download work got done in advance.